lib/ui/pages/news_list/primary_news_list_item.dart

```diff
@@ -0,0 +1,116 @@
+import 'package:flutter/material.dart';
+
+abstract class _Constants {
+  static const double newsImageRadius = 15;
+  static const double newsImageLayerOpacity = 0.7;
+  static const double authorImageRadius = 15;
+  static const double authorRightPadding = 8;
+  static const double stackItemPadding = 20;
+}
+
+class PrimaryNewsListItem extends StatelessWidget {
+  final PrimaryNewsListEntity news;
+
+  PrimaryNewsListItem({Key key, @required this.news})
+      : assert(news != null),
+        super(key: key);
+
+  @override
+  Widget build(BuildContext context) => Stack(
+        children: <Widget>[
+          ClipRRect(
+            borderRadius: BorderRadius.circular(_Constants.newsImageRadius),
+            child: Container(
+              color: Colors.black,
+              child: Opacity(
+                opacity: _Constants.newsImageLayerOpacity,
+                child: Image.network(
+                  news.imageUrl,
+                  //FIXME enter correct dimensions when filling a list with those items
+                  width: 300,
+                  height: 300,
+                  fit: BoxFit.fill,
+                ),
+              ),
+            ),
+          ),
+          Positioned(
+            top: 1,
+            right: 1,
+            child: _buildBookmarkIcon(),
+          ),
+          _buildNewsInfo(context),
+        ],
+      );
+
+  Positioned _buildNewsInfo(BuildContext context) => Positioned(
+        bottom: 1,
+        left: 1,
+        child: Padding(
+          padding: const EdgeInsets.all(_Constants.stackItemPadding),
+          child: Expanded(
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: <Widget>[
+                _buildNewsTitle(context),
+                SizedBox(
+                  height: 15,
+                ),
+                _buildNewsAuthor(context),
+              ],
+            ),
+          ),
+        ),
+      );
+
+  Row _buildNewsAuthor(BuildContext context) => Row(
+        children: <Widget>[
+          Padding(
+            padding: const EdgeInsets.only(right: _Constants.authorRightPadding),
+            child: ClipRRect(
+              borderRadius: BorderRadius.circular(_Constants.authorImageRadius),
+              child: Image.network(
+                news.authorImageUrl,
+                //FIXME enter correct dimensions when filling a list with those items
+                width: 40,
+                height: 40,
+                fit: BoxFit.fill,
+              ),
+            ),
+          ),
+          Text(
+            news.authorName,
+            style: Theme.of(context).textTheme.subtitle2.copyWith(
+                  color: Colors.white,
+                  fontWeight: FontWeight.w300,
+                ),
+          ),
+        ],
+      );
+
+  Container _buildNewsTitle(BuildContext context) => Container(
+        //FIXME enter correct dimensions when filling a list with those items
+        width: 270,
```

Comment:
nie wiem czy trzymałbym się na sztywno szerokości 270 - imo chyba lepiej jakby padding wyznaczał odstęp od krawędzi - wówczas odstęp z jednej i drugiej strony byłby taki sam ;)

Reply:
właśnie nie jestem pewien też szerokości całego itemu, myślałem żeby to ogarnąć jak już będzie się te itemy wrzucało do listy, ale ten title rzeczywiście można ograniczyć samym paddingiem już teraz

Reply:
hmm tu jest problem taki, że żeby text się wrapował to musi być stała szerokość do której ma się wrapować. Próbowałem robić to `FractionalSizedBox` em i niestety bezskutecznie. Ogólnie nad tą częścią najwięcej mi zeszło, może masz jakiś pomysł? Sam padding jak dam to się tekst nie wrapuje

Reply:
zerknę na to, najwyżej jak nic nie uda nam się wymyślić na tę chwilę to się to tak zostawi i kiedyś wróci do tematu ;)

Reply:
jeszcze taki tip, że jak width dasz na jakiś width uzywajac `MediaQuery.of(context).size.width * 0.65` to dziala git

Reply:
najlepiej byloby wyciagnac szerokosc parenta i do niej wrapowanie ograniczyc procentowo

Reply:
jak śmiga to cacy :) jest elastyczne a o to chodziło ;)